src/wallpaper.c

```diff
@@ -179,7 +179,7 @@ static void feh_wm_set_bg_maxed(Pixmap pmap, Imlib_Image im, int use_filelist,
 	gib_imlib_render_image_on_drawable_at_size(pmap, im,
 		render_x, render_y,
 		render_w, render_h,
-		1, 0, !opt.force_aliasing);
+		1, 1, !opt.force_aliasing);
```

Comment:
It may be overkill to have all of these settings changed because I only needed this one (I'm using `--bg-center`), but it seems correct to have the behavior consistent for all of the various flags.